source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs

```diff
@@ -81,6 +81,11 @@ public IActionResult ExportAgreements([FromBody] AcquisitionReportFilterModel fi
             }
 
             var agreements = _acquisitionFileService.SearchAgreements(filter);
+            if (agreements is null || !agreements.Any())
+            {
+                // Return 204 "No Content" to signal the frontend that we did not find any matching records.
+                return NoContent();
```

Comment:
is this coming from the other pr?

Reply:
@devinleighsmith - this change seems to be coming from a different PR. Could you please rebase this branch?